site/content/docs/5.1/components/scrollspy.md

```diff
@@ -252,19 +276,19 @@ Scrollspy also works with `.list-group`s. Scroll the area next to the list group
 </div>
 ```
 
-## Example with simple anchors
+### Simple anchors
 
-Scrollspy is not limited to nav components and list groups, but will work on any `<a>` anchor elements in the current document. Scroll the area and watch the `.active` class change.
+Scrollspy is not limited to nav components and list groups, so it will work on any `<a>` anchor elements in the current document. Scroll the area and watch the `.active` class change.
 
 <div class="bd-example">
   <div class="row">
     <div class="col-4">
-      <div id="simple-list-example" class="d-flex flex-column simple-list-example-scrollspy text-center">
-        <a href="#simple-list-item-1">Item 1</a>
-        <a href="#simple-list-item-2">Item 2</a>
-        <a href="#simple-list-item-3">Item 3</a>
-        <a href="#simple-list-item-4">Item 4</a>
-        <a href="#simple-list-item-5">Item 5</a>
+      <div id="simple-list-example" class="d-flex flex-column gap-2 simple-list-example-scrollspy text-center">
```

Comment:
Weird display IMO + backport style in the code exposed below l.312

Reply:
Let's change it